packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -62,7 +62,7 @@ type InputBoxComponentProps = {
   errorMessage?: string | React.ReactElement;
   disabled?: boolean;
   styles?: InputBoxStylesProps;
-  autoFocus?: 'sendBoxTextField' | false;
+  autoFocus?: 'sendBoxTextField' | 'disabled';
```

Comment:
I don't see this being set to `'disabled'` anywhere.

Reply:
ah I missed that one will also remove like the undefineds above!